docs/setup_git.md

```diff
@@ -466,13 +466,15 @@ Statistics
 
 By Andrew DeOrio <awdeorio@umich.edu>
 
+<!-- primer-spec-highlight-start -->
 ## Quick start
 ```console
 $ make main.exe
```

Comment:
I was expecting something like this:
```
# primer-spec-hightlight begin
$ make main.exe
$ ./main.exe
# primer-spec-highlight end
```

Reply:
Yeah this one’s a bit confusing. We’re actually inside a Markdown code block, hence Markdown/HTML comment syntax applies. 

Also we want to highlight the whole section, including `## Quick Start`. 

---

Aside: This magic comment syntax doesn’t work in `console` blocks since non-console lines are interpreted as “console output” :(

Reply:
Whoops, got it.  Thanks for explaining!